service/src/main/java/bio/terra/workspace/service/workspace/model/JobStateFilter.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.workspace.service.workspace.model;
+
+import bio.terra.workspace.generated.model.ApiJobStateFilter;
+
+public enum JobStateFilter {
+  ALL(ApiJobStateFilter.ALL),
```

Comment:
what does ALL mean?

Reply:
I added some doc'n to the file.

Reply:
where is the doc?

Reply:
Huh. It is there when I look at the code in the branch and it is there on my machine. I wonder why you are not seeing it. Here is the link:
https://github.com/DataBiosphere/terra-workspace-manager/blob/8fc595e458f6f1d265b7b0ddac617da5d2c0942c/service/src/main/java/bio/terra/workspace/service/workspace/model/JobStateFilter.java#L7